integration-tests/test_client.py

```diff
@@ -0,0 +1,99 @@
+import glob
+import logging
+import os
+import subprocess
+from os.path import exists
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_client_files_permission(insights_client):
+    """Verify that permission for following files is 0644:
+    - /etc/insights-client/.lastupload
+    - /etc/insights-client/.last-upload.results only created in legacy mode which
+    will be deprecated soon, hence skipping
+
+    Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1924990
+    """
+    file_ = "/etc/insights-client/.lastupload"
```

Comment:
rather than `file_`, i'd give it a better name e.g. `last_upload_file`

Reply:
I vote for the same - ie. it is better to recognize the main meaning of it.